index.html

```diff
@@ -19,10 +20,14 @@
         </a>
       </div>
       <nav>
-        <label for="nav-check" id="hamburger-button"><i class="fa fa-bars"></i></label>
-        <input id="nav-check" name="nav-check" type="checkbox">
+        <label for="nav-check" id="hamburger-button"
+          ><i class="fa fa-bars"></i
+        ></label>
+        <input id="nav-check" name="nav-check" type="checkbox" />
         <ul class="nav-item-container">
-          <label for="nav-check" id="mobile-close-btn"><i class="fa fa-times" aria-hidden="true"></i></label>
+          <label for="nav-check" id="mobile-close-btn"
+            ><i class="fa fa-times" aria-hidden="true"></i
```

Comment:
what is `aria-hidden`?

Reply:
aria-hidden removes the element from accessibility tree means it tells the screen reader to ignore it